### PR TITLE
Feature/information component

### PR DESCRIPTION
**Pull request by dewarian**
Created 'information component' with the necessary documentation according to the format structure of components.

Review and comment if changes are neede.

### CODE REVIEW

File: components/information/README.md [hunk 1]
```diff
@@ -0,0 +1,43 @@
+# Information component
+
+## Table of content
+- [What does it do]()
```
Comment (sjagoori) [inline components/information/README.md:4]: What it does* and also add the # so we can click on em

File: components/information/README.md [hunk 2]
```diff
@@ -0,0 +1,43 @@
...
+
+## Developer
+[Nathan Bommezijn](github.com/dewarian)
+
+## Changelog
+
+All notable changes to this component will be documented in this file.
+The format adheres to the template from [Shabier](github.com/sjagoori) and [Keep a changelog](https://keepachangelog.com/en/1.0.0/).
+
+## 1.0.0 - 2021-01-14
```
Comment (sjagoori) [inline components/information/README.md:39]: remove date, we have git for history. use [this](https://github.com/sjagoori/KNMI/edit/master/components/button/README.md) as template

File: components/information/information.js [hunk 3]
```diff
@@ -0,0 +1,63 @@
+import { Component } from 'react'
```
Comment (sjagoori) [inline components/information/information.js:1]: Fix the indents in this file, we're using 2 spaces

File: components/information/information.js [hunk 4]
```diff
@@ -0,0 +1,63 @@
...
+                        <Image src={this.props.imgUrl} />
+                    </ImageContainer>
+                </Container>
+            </ThemeProvider>
+        )
+    }
+}
+
+const Container = styled.div`
+background-color: ${props => props.theme.colors.white};
```
Comment (sjagoori) [inline components/information/information.js:29]: indent

File: pages/index.js [hunk 5]
```diff
@@ -1,3 +1,4 @@
+import Information from '@/components/information/information'
```
Comment (sjagoori) [inline pages/index.js:1]: the `@/` in the import is unnecessary, the [jsconfig](https://github.com/sjagoori/KNMI/blob/master/jsconfig.json) resolves that.

i noticed that you removed code from the index, you can remove this line if this import isn't used

**Review by sjagoori — CHANGES_REQUESTED** [review]
Good stuff! See, it's easier than expected. You got this!

File: components/information/README.md (hunk 1 above)
Comment (dewarian) [reply components/information/README.md:4]: Filled the links.

File: components/information/information.js (hunk 3 above)
Comment (dewarian) [reply components/information/information.js:1]: This was one hell of a job, changed tab and indent settings. Changed indents to 2 spaces.

File: pages/index.js (hunk 5 above)
Comment (dewarian) [reply pages/index.js:1]: Accidentally left this in. Is removed in the next version.

File: components/information/information.js (hunk 3 above)
Comment (sjagoori) [reply components/information/information.js:1]: Sir it's still 4 spaces, have you tried formatting the file after setting it to 2 spaces?

**Review by sjagoori — CHANGES_REQUESTED** [review]

**Comment by dewarian** [comment]
For history: Link to the revision [#828dca2](https://github.com/sjagoori/KNMI/commit/828dca2775ecaf99ae1d4e4c8acfd4c2808dbc86).

**Review by sjagoori — APPROVED** [review]

**Comment by sjagoori** [comment]
image

**Pull request merged**